### PR TITLE
Add Getting Started and Distributions docs

### LINKED ISSUE #1: Provide separate Modelers
**`base`**

* [x] diagram-js-minimap
* [x] diagram-js-origin & @bpmn-io/align-to-origin (exclusive, optional)
  * Default: align-to-origin
  * config.disableAdjustOrigin: diagram-js-origin
* [ ] <del>@bpmn-io/add-exporter</del> --> this would require to set exporter metadata any time
* [x] bpmn-js-executable-fix
* [x] bpmn-js-signavio-compat
* [x] bpmn-js-disable-collapsed-subprocess
* [x] bpmn-js-properties-panel
* [x] BPMN properties provider

**`camunda-cloud`**
cf. [`zeebe-io/zeebe-modeler#BpmnModeler`](https://github.com/zeebe-io/zeebe-modeler/blob/develop/client/src/app/tabs/bpmn/modeler/BpmnModeler.js)

Everything from `base` + zeebe related modules
* [x] context-pad
* [x] palette
* [x] modeling
* [x] replace-menu
* [x] rules
* [x] zeebe-bpmn-moddle
* [x] zeebe-properties-provider (implemented, tests are missing)

**`camunda-platform`**
cf. [`camunda/camunda-modeler#BpmnModeler`](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/bpmn/modeler/BpmnModeler.js)

Everything from `base` + camunda related modules
* [x] camunda-bpmn-moddle
* [x] camunda-properties-provider

**What else?**

available in Camunda Modeler as well, do we need them here? --> I'd vote for no
* [ ] <del>complete-direct-editing</del>
* [ ] <del>global-clipboard</del>
* [ ] <del>hand-tool-on-space</del>
* [ ] <del>properties-panel-keyboard-bindings</del>

-----------------

Related to https://github.com/zeebe-io/zeebe-modeler/issues/288

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #1
Related to https://github.com/zeebe-io/zeebe-modeler/issues/288

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
@camunda/modeling-dev any chance to get a review here?

File: README.md [hunk 1]
```diff
@@ -3,17 +3,80 @@
...
+
+or include it via npm
+
+```js
+import BpmnModeler from 'camunda-bpmn-js/lib/camunda-platform/Modeler';
+
+import 'camunda-bpmn-js/dist/assets/camunda-platform-modeler.css';
+```
+
+into your node-style web-application.
```
Comment (barmac) [inline README.md:28]: Why is it _node-style_?

Comment (pinussilvestrus) [reply README.md:28]: Like in a basic node application via `import`.

Copied it from the [`bpmn-js` introduction](https://github.com/bpmn-io/bpmn-js#installation). Maybe we could restructure it to make it clear "node-style" only refers to the second possibility.

Comment (barmac) [reply README.md:28]: If I understand it correctly, by _node-style web-application_ we meant applications that are bundled. Is that correct? 

Anyway, I would say that you can _use pre-packaged distributions or install it via npm_. Most app starters like create-react-app already include webpack or rollup setup, so it should work even if don't know anything about bundling.

Comment (barmac) [reply README.md:28]: or we can leave it as it is

**Review by barmac — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pinussilvestrus) [reply README.md:28]: > use pre-packaged distributions or install it via npm

Let's go for that. Thanks :+1:

**Pull request merged**